apps/site/src/app/(home)/sections/About/About.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from "next/image";
+import Lantern from "@/lib/components/Lantern/Lantern";
+import koiJump from "@/assets/images/koi-jump.png";
+import styles from "./About.module.css";
+
+const About = () => {
+	return (
+		<>
+			<section className="container md mx-auto text-center my-14 w-3/4 sm:w-1/2 xl:w-2/5 relative">
+				<Lantern
+					width={300}
+					height={300}
+					className="lg:hidden mx-auto"
+				/>
+				<Lantern
+					width={300}
+					height={300}
+					className="absolute hidden lg:block"
+					style={{ top: "30%", left: "-45%" }}
+				/>
+				<Lantern
+					width={350}
+					height={350}
+					className="absolute hidden lg:block"
+					style={{ top: "75%", right: "-50%" }}
+				/>
+				<div className="text-2xl lg:text-3xl mb-5">
+					<h2>
+						<span className={styles.statistic}>36</span> Hours
+					</h2>
+					<h2>
+						<span className={styles.statistic}>500+</span> Hackers
+					</h2>
+					<h2>
+						<span className={styles.statistic}>&#36;10,000</span> in
+						Prizes
+					</h2>
+				</div>
+				<h3 className="text-xl lg:text-2xl mb-5">
+					Create &#43; Connect &#43; Inspire
```

Comment:
Question: is there a particular reason the plus sign needs to be escaped?

Reply:
I thought this and the other symbols had to be escaped, like how we did with the quotes. I'll unescape them.